contrib/runners/winrm_runner/BUILD

```diff
@@ -5,3 +5,18 @@ python_requirement(
     # https://github.com/pantsbuild/pants/pull/17390
     modules=["winrm"],
 )
+
+st2_runner_python_distribution(
+    runner_name="winrm",
+    description=(
+        "WinRM shell command and PowerShell script action runner "
```

Comment:
```suggestion
        "WinRM shell command and PowerShell command and script action runner "
```

Reply:
Good catch! I just copied it from here:

https://github.com/StackStorm/st2/blob/9d311c501b47c3055447832ede2809da9c5b6855/contrib/runners/winrm_runner/setup.py#L37-L40